docs/modding/suitmodding/addinglocals.mdx

```diff
@@ -0,0 +1,666 @@
+
+### 2. Convert your string
+- Paste your string inside that box
+- Copy *only* the converted <Tooltip text={<strong>decimal number</strong>} tooltip="A string converted into a numeral form" />. 
+
+>*We now have everything needed for the next part:*
+ >- a <Tooltip text={<strong>decimal form</strong>} tooltip="A string converted into a numeral form" /> of our <Tooltip text={<strong>string</strong>} tooltip= "A key that links to text in the LOCR File" />
+ >- the <Tooltip text={<strong>HASH</strong>} tooltip="A unique identifier that acts as the address or coordinates for a file." /> from the <Tooltip text={<strong>LOCR File</strong>} tooltip="A file that holds game text in different languages, linked by unique 'strings'." />
+
+:::tip windows clipboard
+Easily manage multiple copied items using <Tooltip text={<strong>Windows clipboard</strong>} tooltip="A built-in tool that stores multiple copied items." />. Press `Windows + V` to activate it. 
+
+Now copy both your: 
+- LOCR's <Tooltip text={<strong>HASH</strong>} tooltip="A unique identifier that acts as the address or coordinates for a file." /> and 
+- <Tooltip text={<strong>decimal</strong>} tooltip="A string converted into a numeral form" />.
+>Make it a habit to paste using `Windows + V` instead of `Ctrl + V` from now on. 
+
+ *It might seem overkill for 2 lines of codes but it'll pay off and it beats* *<Tooltip text={<italic>using notepad</italic>} tooltip="💀" />*  
+ 
+:::
+
+---
+
+## Pt. 4: **UPDATE MANIFEST**
+*Here we will be adding* <Tooltip text={<><strong>localisation override code</strong><span className="icon-alert"></span></>} tooltip="A way to replace or modify in-game text, like names or descriptions, with your own custom text. It allows you to customize how text appears without altering the core game files." /> *to your manifest file. It will contain:*
+- <Tooltip text={<strong>HASH</strong>} tooltip="A unique identifier that acts as the address or coordinates for a file." /> of your <Tooltip text={<strong>LOCR File</strong>} tooltip="A file that holds game text in different languages, linked by unique 'strings'." />
+- <Tooltip text={<strong>decimal form</strong>} tooltip="A string converted into a numeral form" /> of your <Tooltip text={<strong>string</strong>} tooltip= "A key that links to text in the LOCR File" />
+- *Your new text*
+
+### 1. Add the localisation override line 
+
+- Open your manifest
+- Add this line: `"localisationOverrides": {}`
+
+<details>
+
+<summary>How-to Manifest</summary>
+
+Theres *2 methods* to doing this:
+- *Method 1*: 
+    - just copying: `"localisationOverrides": {}`
+>Or
+- *Method 2*:
+    - manually typing it in, starting with `"`
+    - navigate the *suggestions menu* to find `localisationOverrides`.
+    
```

Comment:
This is just basic text editing, and I feel like if they cannot distinguish between copy/paste and manual typing there will be other challenges before they get to this point

Reply:
Trying to find a sweet spot for noobs and intermediate users was the balance I tried to keep. Which is why there is also weird distinguishing terms like file path and file name.